ui/com/composer/index.jsx

```diff
@@ -214,10 +379,15 @@ export default class Composer extends React.Component {
     mentionslib.extract(text, (err, mentions) => {
```

Comment:
by the way, this is repeated a lot in Editor also (prepping the text to send), but I'm not quite sure how would be best to combine yet, since sending goes on in the callback.